CONTRIBUTING.md

```diff
@@ -52,9 +54,32 @@ export class HelloWorldCommand extends Command {
 module.exports = [HelloWorldCommand]
 ```
 
-Lastly, test files must be created in the `__tests__/` folder. `jest` is used to run the tests and a CI has been set using GitHub Actions to ensure all tests are passing when merging a Pull Request.
+Lastly, unit tests must be created in the `__tests__/` folder. The tests can then be launched with the `yarn test` command: it will find all files with a filename ending in `.test.ts` in the repo and execute them.
+
+#### Beta command
+
+If your command is related to a beta product or feature, or that you want to dogfood the command first, you can mark you command as beta.
+
+To do so, add your command's name to the [`BETA_COMMANDS` array](https://github.com/DataDog/datadog-ci/blob/35c54e1d1e991d21461084ef2e346ca1c6bb7ea6/src/cli.ts#L8).
+
+Users will have to prefix their command line with `DD_BETA_COMMANDS_ENABLED=1`. Please document this in your command's README for visibility. This will be something to remove once the command goes out of beta.
```

Comment:
```suggestion
Users have to prefix their command line with `DD_BETA_COMMANDS_ENABLED=1` to use the command. Make sure to document this in your command's README for visibility. This is removed once the command goes out of beta.
```

Reply:
@maycmlee what about "This _should be removed_ once the command goes out of beta"?